csharp/src/Google.Protobuf/WritingPrimitives.cs

```diff
@@ -208,6 +213,99 @@ public static void WriteString(ref Span<byte> buffer, ref WriterInternalState st
             }
         }
 
+        private static void WriteAsciiStringToBuffer(Span<byte> buffer, ref WriterInternalState state, string value, int length)
```

Comment:
nit: add a comment that we're assuming that  we've previously checked that the string contains only ascii characters and thus the number of resulting bytes will the the same as the number of input characters.

Reply:
Done